### DISCUSSION TITLE
Unable to add new Apps or Manually run new builds

### DISCUSSION THREAD

**Discussion by altmas5** [codemagic-ci-cd/codemagic-docs#2093] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Android (Kotlin)

### Steps to reproduce

1. Click on the `Start new build` button
2.  Wait for the list of branches to be pulled

### Expected results

Get the list of branches and workflows

### Actual results

The load never ends
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/1054618/ab0a4c57-c728-45c1-a7e5-c3ebe59621fb)

**Comment by zuhriyansauqi** [comment]
<img width="1118" alt="Screenshot 2023-10-06 at 05 42 28" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/4607523/d5858ca9-ac25-4c6a-bc0e-3660fc8bca85">

I also encountered an issue where the codemagic.yaml file isn't visible in the app settings.

**Comment by appsowner** [comment]
Hello,

same symptom

any solution ?

Thanks,

**Comment by gonzalogauto** [comment]
Seems that this discussion is duplicated. Same discussion here #2092
